beacon_node/lighthouse_network/src/rpc/protocol.rs

```diff
@@ -272,11 +303,8 @@ impl RpcLimits {
 /// Tracks the types in a protocol id.
 #[derive(Clone, Debug)]
 pub struct ProtocolId {
-    /// The RPC message type/name.
-    pub message_name: Protocol,
-
-    /// The version of the RPC.
-    pub version: Version,
+    /// The protocol name and version
+    pub protocol: SupportedProtocol,
```

Comment:
to make this a bit less awkward we could maybe use `versioned_protocol` ?

So that access is a bit more clear in cases like `request.protocol.protocol()` -> `request.versioned_protocol.protocol()`

It's more verbose but I find it clearer. Thoughts?

Reply:
Yes it's a lot more clear, thanks for the suggestion :)